client/src/components/BookWrapper/BookWrapper.js

```diff
@@ -26,15 +26,29 @@ class BookWrapper extends Component {
               <h3 className="sub-title">Dates</h3>
               <div className="book-subcontainer">
                 <div className="book-checkin-container">
-                  <h3 className="book-checkin" id="checkin">
+                  <h3
+                    className={
```

Comment:
checkout this this might be helpfull with the classes

https://www.npmjs.com/package/classnames